event-internal.h

```diff
@@ -368,7 +368,7 @@ struct event_config {
 };
 
 /* Internal use only: Functions that might be missing from <sys/queue.h> */
-#if defined(EVENT__HAVE_SYS_QUEUE_H) && !defined(EVENT__HAVE_TAILQFOREACH)
```

Comment:
I'm not even sure why we do that extra check,

#ifndef TAILQ_WHATEVER
#define TAILQ_WHATEVER(...)
#endif

Should be fine.